app/lib/mappers/mapGpCounts.js

```diff
@@ -0,0 +1,30 @@
+const xmlJsonHelper = require('../xmlJsonHelper');
+
+const GP_COUNT = 's:gpcount';
+
+function countSectionValid(rawCounts) {
+  return rawCounts && rawCounts[GP_COUNT];
+}
+
+function mapGpCount(gpCount, rawGpCount) {
+  if (rawGpCount.$ && rawGpCount.$.type) {
+    // eslint-disable-next-line no-param-reassign
+    gpCount[rawGpCount.$.type] = Number(rawGpCount._);
+  }
+  return gpCount;
+}
+
+function emptyObjectToUndefined(gpCounts) {
+  return Object.keys(gpCounts).length > 0 ? gpCounts : undefined;
```

Comment:
This looks like a good candidate to go in a `utils` module although with only being used once it might be a bit  early to do that...